chrome/content/zotero/xpcom/storage/storageLocal.js

```diff
@@ -21,6 +21,7 @@ Zotero.Sync.Storage.Local = {
 		switch (libraryType) {
 		case 'user':
 		case 'publications':
+		case 'feed':
```

Comment:
I don't think this should be here — if this is getting called on feed libraries during sync, that's a bug elsewhere that should be fixed.